### PR TITLE
removes a direct use of Destroy()

### PR DESCRIPTION
**Pull request by ghost**
:cl:
 code: removes a direct use of Destroy()
/:cl:

pkp thats not kosher man

### CODE REVIEW

File: code/datums/mutations/antenna.dm [hunk 1]
```diff
@@ -28,7 +28,7 @@
 	if(..())
 		return
 	if(linked_radio)
-		linked_radio.Destroy()
+		qdel(linked_radio)
```
Comment (Cyberboss) [inline code/datums/mutations/antenna.dm:31]: My dude
```suggestion
		QDEL_NULL(linked_radio)
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Qustinnus** [comment]
whatever nerd 🛹

File: code/datums/mutations/antenna.dm [hunk 2]
```diff
@@ -28,7 +28,7 @@
 	if(..())
 		return
 	if(linked_radio)
```
Comment (spookydonut) [inline code/datums/mutations/antenna.dm:30]:
```suggestion
```

File: code/datums/mutations/antenna.dm [hunk 3]
```diff
@@ -28,7 +28,7 @@
...
-		linked_radio.Destroy()
+		QDEL_NULL(linked_radio)
```
Comment (spookydonut) [inline code/datums/mutations/antenna.dm:31]:
```suggestion
	QDEL_NULL(linked_radio)
```

**Review by spookydonut — CHANGES_REQUESTED** [review]

File: code/datums/mutations/antenna.dm (hunk 3 above)
Comment (Jared-Fogle) [reply code/datums/mutations/antenna.dm:31]: you can make multi line suggestions you know

Comment (spookydonut) [reply code/datums/mutations/antenna.dm:31]: you cant for lines that were deleted, try it.

Comment (Jared-Fogle) [reply code/datums/mutations/antenna.dm:31]: fair enough

**Pull request merged**